### DISCUSSION TITLE
Class to manage multi-agents

### DISCUSSION THREAD

**Discussion by andreibondarev** [patterns-ai-core/langchainrb#865] [Ideas]
Similar to the CrewAI's Crews or OpenAI Swarm's we'd like to figure out a way to combine agents that can communicate with each other and accomplish tasks together.

**Comment by benguillet** [comment]
Any plans to port langgraph sdk to ruby? Could not find any mention of it. That would allow creating a graph of agents, with shared memory etc

**Comment by luizcarvalho** [comment]
I'm building a system to manage multiple agents, but without communication between them
